cluster-operator/src/main/java/io/strimzi/operator/cluster/model/RackConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RackConfig {
+
+    private String topologyKey;
+
+    public RackConfig() {
+
+    }
+
+    public RackConfig(String topologyKey) {
+        this.topologyKey = topologyKey;
+    }
+
+    @JsonProperty("topologyKey")
```

Comment:
Since the getter name matches the JSON key this annotation should be unnecessary in this case.